### PR TITLE
[10.0] mig account_due_list

### PR DESCRIPTION
**Pull request by scigghia**

### CODE REVIEW

**Comment by scigghia** [comment]
[WIP]

**Comment by scigghia** [comment]
i think the module is usable, someone could check?
p.s. @pedrobaeza the field author in manifest is too short, can I change "Serv. Tecnol. Avanzados - Pedro M. Baeza" with "Pedro.zip" =) or "Tecnativa"?

**Comment by pedrobaeza** [comment]
Please change it to `Tecnativa`

**Comment by scigghia** [comment]
done

**Comment by scigghia** [comment]
need review

File: account_due_list/migrations/8.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    ('account_move_line', 'day'),
+}
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    if not version:
+        return
+
+    openupgrade.drop_columns(cr, column_drops)
```
Comment (eLBati) [inline account_due_list/migrations/8.0.1.0.0/pre-migration.py]: @scigghia this file should be removed

**Comment by eLBati** [comment]
@scigghia please keep an eye on https://github.com/OCA/account-payment/pull/101 also

File: account_due_list/migrations/8.0.1.0.0/pre-migration.py (hunk 1 above)
Comment (scigghia) [reply account_due_list/migrations/8.0.1.0.0/pre-migration.py]: sure

**Comment by pedrobaeza** [comment]
I have tried #101 and changes too much things that I don't like:
- Two separate menus for customer/suppliers.
- Not all receivable nor payable accounts are displayed.

File: account_due_list/views/payment_view.xml [hunk 2]
```diff
@@ -1,15 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
```
Comment (susport) [inline account_due_list/views/payment_view.xml:2]: odoo by openerp, OCA guide.

File: account_due_list/models/account_move_line.py [hunk 3]
```diff
@@ -1,87 +1,80 @@
...
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 #
 ##############################################################################
 
-from openerp.tools.translate import _
-from openerp import models, fields, api
-from openerp.osv import orm
+from odoo import models, fields, api, _
```
Comment (susport) [inline account_due_list/models/account_move_line.py:17]: Order by name the import. OCA guide

**Review by eLBati — CHANGES_REQUESTED** [review]
@scigghia Please have a look at my comments. What do you think? Thanks!

**Comment by JordiBForgeFlow** [comment]
Perhaps we shall merge first https://github.com/OCA/account-payment/pull/123, and then use the 9.0 branch as the basis to migrate to 10.0. What do you think @scigghia @eLBati ?

**Comment by scigghia** [comment]
I'll take a look asap
I agree with @eLBati  and i wan to check @jbeficent to test #123 before manage this PR
sorry for lack of time

**Comment by eLBati** [comment]
To be thorough, an open question
https://github.com/OCA/account-payment/pull/122#pullrequestreview-23447612
on '[9.0] [MIG] account_due_list'

**Comment by pedrobaeza** [comment]
We have made several changes to the final 9.0 branch. Can you rebase on top of it?

**Comment by vonpupp** [comment]
In order to contribute and learn more I did the rebase on top of 9 on [my fork] based on this PR.

Since it is the first time I contribute within a PR which is not mine, I don't know if it is possible to add the modifications to this PR. Is it?

[my fork]: https://github.com/vonpupp/account-payment/tree/scigghia-pr

**Comment by pedrobaeza** [comment]
You can make a PR over the PR branch (scigghia/10.0-mig-account_due_list)

**Comment by eLBati** [comment]
Closing this and moving to https://github.com/OCA/account-payment/pull/129

**Pull request closed without merging**